week-1/Homework/mandatory/2-logic-error.js

```diff
@@ -1,31 +1,31 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+    return word;
```

Comment:
In this function you're supposed to find a method that trims the string (removes whitespace from both ends of it). For example:
`   hanif    ` would became `hanif`

You don't need to do it yourself, JavaScript has built in methods that you can use on strings. You can try looking for it on google and implement it here